web-client/public/gene/api.js

```diff
@@ -178,18 +178,28 @@
             };
 
             var parseUniprot = function (data) {
-                return {
+                var uniprotTemplate = {
                     uniprotID: XMLParser(data.getElementsByTagName("name")[0]),
                     proteinSequence: XMLParser(data.getElementsByTagName("sequence")[0]),
                     proteinType: XMLParser(data.getElementsByTagName("protein")[0].childNodes[1].childNodes[1]),
                     species: XMLParser(data.getElementsByTagName("organism")[0].childNodes[1]),
                 };
+
+                for (var prop in uniprotTemplate) {
+                    if ((uniprotTemplate[prop] === undefined) ||
+                    (uniprotTemplate[prop] === null)) {
```

Comment:
Typically when a line break occurs mid-expression, the follow-up line is indented to indicate that the thought isn't "complete" yet. Alternatively, this particular expression doesn't actually exceed our 120-character max length yet, so can just be formatted as a single line.